src/pages/progress_reports/DarCloseout.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { FormField, FormFieldTypes } from '../../components/forms/forms';
+
+interface FormStateInterface {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any;
+}
+
+interface DarCloseoutProps {
+    readonly state: string;
+}
+
+interface FormFieldChange {
+    key: string;
+    value: boolean | string;
+}
+
+export default function DarCloseout(_props: DarCloseoutProps): React.JSX.Element {
```

Comment:
Where is this component used? I don't see routing or a usage in another component?

Reply:
Yes, the component will be used in the main progress report form, which Shelby is working on.